core-services/prow/02_config/_config.yaml

```diff
@@ -1106,6 +1107,18 @@ tide:
     - bugzilla/invalid-bug
     repos:
     - openshift/sriov-tests
+  - labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/blocked-paths
+    - bugzilla/invalid-bug
+    repos:
+    - openshift/network-metrics-daemon
```

Comment:
Please add this to the group of repos in L725-L893, there is no need to start your own group if you have the same label requirements

Reply:
@alvaroaleman thanks for the comment done, can you please take another look at the PR?